graphql/resolvers.go

```diff
@@ -965,6 +965,44 @@ func (r *queryResolver) ProjectSettings(ctx context.Context, identifier string)
 	return res, nil
 }
 
+func (r *mutationResolver) CreateProject(ctx context.Context, project restModel.APIProjectRef) (*restModel.APIProjectRef, error) {
+	projectRef, err := model.FindOneProjectRef(*project.Identifier)
+	if err != nil && err.(gimlet.ErrorResponse).StatusCode != http.StatusNotFound {
+		return nil, InternalServerError.Send(ctx, fmt.Sprintf("error looking in project collection: %s", err.Error()))
+	}
+	if projectRef != nil {
+		return nil, InputValidationError.Send(ctx, fmt.Sprintf("cannot create project with identifier '%s'", err.Error()))
```

Comment:
That touches on a theoretical conversation of how much information to give away. If we do that, a user can technically hit this a bunch of times with different identifiers to find out what already exists.

Reply:
They can see the same information by visiting the waterfall and clicking through projects Or simply requesting the projects endpoint. I don't think we have the concept of hidden projects in evergreen. So it is probably ok to expose this. Also this endpoint is going to be used by admins only and gated as such, So it should be ok to expose this information.

Reply:
We do have a concept of hidden projects on Evergreen; that being said, the fear of people learning too much by being told that an identifier exists already was a Sam fear from before our permissions scheme was improved; now even if they had a hidden project identifier they wouldn't have the access to use it.

Reply:
> Or simply requesting the projects endpoint
Which endpoint are you referring to? How would them requesting the endpoint give them that information?

I'm okay with either, but I hear the argument for both. However, maybe this decision should be a broader conversation so that the whole team can all follow the same convention? @ablack12 I think this should ultimately be your decision.

Reply:
This truly doesn't come up often so I don't think it's worth a full team conversation (the only change would really just be to update the legacy which we don't really need to do). We may as well change it to be more verbose; people do sometimes ask us about this.

Reply:
yeah, that makes sense! I'll make it more verbose.